runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -24,32 +24,44 @@
  */
 package net.runelite.client.plugins.antidrag;
 
-import com.google.inject.Provides;
-import java.awt.event.KeyEvent;
+import lombok.extern.slf4j.Slf4j;
 import javax.inject.Inject;
-import net.runelite.api.Client;
-import net.runelite.api.events.FocusChanged;
+import java.awt.event.KeyEvent;
+import com.google.inject.Provides;
+import net.runelite.api.GameState;
+import net.runelite.api.events.WidgetLoaded;
 import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetID;
 import net.runelite.api.widgets.WidgetInfo;
-import net.runelite.client.config.ConfigManager;
+import net.runelite.client.callback.ClientThread;
 import net.runelite.client.eventbus.Subscribe;
+import net.runelite.api.Client;
+import net.runelite.api.events.VarbitChanged;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.events.ConfigChanged;
 import net.runelite.client.input.KeyListener;
 import net.runelite.client.input.KeyManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 
+
 @PluginDescriptor(
-	name = "Shift Anti Drag",
-	description = "Prevent dragging an item for a specified delay",
-	tags = {"antidrag", "delay", "inventory", "items"}
+	name = "Anti Drag",
+	enabledByDefault = false
 )
+@Slf4j
 public class AntiDragPlugin extends Plugin implements KeyListener
 {
-	private static final int DEFAULT_DELAY = 5;
+	static final String CONFIG_GROUP = "antiDrag";
```

Comment:
I'm sorry I am still trying to learn my way around github and understand the review process. Are you referring to line 55 - `static final String CONFIG_GROUP = "antiDrag";`? If so, are you suggesting referencing the `ConfigGroup`
via `AntiDragConifg.class.getAnnotation(ConfigGroup.class).value()`?

Reply:
He means that `AntiDragConfig` should also have its `@ConfigGroup` definition updated to reference this constant. So it would read like this:

```java
@ConfigGroup(AntiDragPlugin.CONFIG_GROUP)
```